src/sql/azdata.proposed.d.ts

```diff
@@ -305,6 +306,41 @@ declare module 'azdata' {
 		required?: boolean;
 	}
 
+	/**
+	 * A property to be displayed in the PropertiesContainerComponent
+	 */
+	export interface DisplayProperty {
```

Comment:
this is only related to the PropertiesContainer, maybe naming it PropertiesContainerItem?